go.mod

```diff
@@ -122,7 +122,7 @@ require (
 )
 
 replace (
-	github.com/gophercloud/gophercloud => /home/xagent/gophercloud
-	github.com/gophercloud/utils => /home/xagent/utils
-	k8s.io/cloud-provider => /home/xagent/k8s-cloud-provider
+	github.com/gophercloud/gophercloud => ../gophercloud
```

Comment:
should we tie them to a version ?

Reply:
we dont have to worry about checking it out seperately every time we try to build them, unless we have some custom changes in gophercloud.

Reply:
I am not sure why it is done this way. But I will come to this in a separate PR if it is not critical.